vector/src/main/java/lt/lrv/basemap/layers/Transportation.java

```diff
@@ -86,13 +86,11 @@ public void addTransportationFeature(String clazz, String subclass, int minZoom,
                 .setBufferPixels(BUFFER_SIZE)
                 .setAttr(Fields.CLASS, clazz)
                 .setAttr(Fields.SUBCLASS, subclass)
-                .setAttr(Fields.EXPRESSWAY, expressway)
-                .setAttr(Fields.LEVEL, level)
```

Comment:
I suggest to leave level as we have now because it will help to solve multilevel road intersections. Right now style doesn't support that but we can implement that by adding additional casing for 2-3 level roads.